99_Appendices/F_Debugging.md

```diff
@@ -212,19 +212,30 @@ then on another shell, on the same folder where we started the emulator launch t
 socat -,echo=0,icanon=0 unix-connect:qemu-monitor-socket
 ```
 
-This will prompt with a shell similar to the following:
+
+Another method is to use telnet to start the monitor. This is handy for easier, cross-platform or remote use (albeit less secure).
+
+```bash
+qemu-system-i386 [..other params..] -monitor telnet::45454,server,nowait
+```
+
+This enables the monitor to listen on a specified port (ie, 45454). You can then connect to the QEMU monitor from another terminal or a remote machine (with networking setup) using Telnet:
```

Comment:
i see telnet here with capital T, and in the previous sentence with lowercase t, maybe we want to use the same capitalization on both? Or is there a reason?

Reply:
Thank you, also fixed some to do with Qemu, didn't know if there were more